src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -201,26 +185,10 @@ public static Change genericTechChange(final TechAttachment attachment, final Bo
     return new GenericTechChange(attachment, value, property);
   }
 
-  public static Change changeGameSteps(final GameSequence oldSequence, final GameStep[] newSteps) {
-    return new GameSequenceChange(oldSequence, newSteps);
-  }
-
   public static Change unitPropertyChange(final Unit unit, final Object newValue, final String propertyName) {
     return new ObjectPropertyChange(unit, propertyName, newValue);
   }
 
-  public static Change addAttachmentChange(final IAttachment attachment, final Attachable attachable,
```

Comment:
I am not sure if this one (`addAttachmentChange` and `AddAttachmentChange` and `RemoveAttachmentChange`) should be kept or not. This may be good functionality to make available to map makers? Or it may introduce too many problems to let people start using it...